ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/ScanConfigs/Wizard/ProfileSelection.tsx

```diff
@@ -49,61 +69,89 @@ function ProfileSelection({ profiles, isFetchingProfiles }: ProfileSelectionProp
             .filter((_, index) => {
                 return index === rowId ? isSelected : selected[index];
             })
-            .map((profile) => profile.id);
+            .map((profile) => profile.name);
 
         setFieldValue('profiles', newSelectedIds);
     };
 
     const handleSelectAll = (event: React.FormEvent<HTMLInputElement>, isSelected: boolean) => {
         onSelectAll(event, isSelected);
 
-        const newSelectedIds = isSelected ? profiles.map((profile) => profile.id) : [];
+        const newSelectedIds = isSelected ? profiles.map((profile) => profile.name) : [];
 
         setFieldValue('profiles', newSelectedIds);
     };
 
+    const isProfileExpanded = (name: string) => expandedProfileNames.includes(name);
+
     function renderTableContent() {
-        return profiles?.map(({ id, name, description }, rowIndex) => (
-            <Tr key={id}>
-                <Td
-                    key={id}
-                    select={{
-                        rowIndex,
-                        onSelect: (event, isSelected) => handleSelect(event, isSelected, rowIndex),
-                        isSelected: selected[rowIndex],
-                    }}
-                />
-                <Td>{name}</Td>
-                <Td>{description}</Td>
-            </Tr>
+        return profiles?.map(({ description, name, productType, ruleCount, title }, rowIndex) => (
+            <Tbody>
+                <Tr key={name}>
+                    <Td
+                        key={name}
+                        select={{
+                            rowIndex,
+                            onSelect: (event, isSelected) =>
+                                handleSelect(event, isSelected, rowIndex),
+                            isSelected: selected[rowIndex],
+                        }}
+                    />
+                    <Td
+                        expand={{
+                            rowIndex,
+                            isExpanded: isProfileExpanded(name),
+                            onToggle: () => setProfileExpanded(name, !isProfileExpanded(name)),
+                        }}
+                    />
+                    <Td>{name}</Td>
+                    <Td>{ruleCount}</Td>
+                    <Td>{productType}</Td>
+                </Tr>
+                <Tr isExpanded={isProfileExpanded(name)}>
+                    <Td colSpan={1}></Td>
+                    <Td colSpan={1}></Td>
+                    <Td dataLabel="" colSpan={3}>
+                        <ExpandableRowContent>
+                            <Text className="pf-u-font-weight-bold">{title}</Text>
+                            <Divider component="div" className="pf-u-my-md" />
+                            <Text>{description}</Text>
+                        </ExpandableRowContent>
+                    </Td>
```

Comment:
I think the dataLabel ends up being linked to the cell content for accessibility. If we're making it blank, I think the td needs some kind of aria attribute with the th content.

Reply:
At least one purpose of `dataLabel` prop is for change at minimum width from table to label value pairs.

@bradr5 can you reduce width of page to confirm benefit of `dataLabel` even if `Th` element is empty?

Reply:
Oh, this is in the expandable row. A picture will be interesting to see if ordinary principle applies.

Reply:
Ah, that would explain issue found in the bug bash that I haven't looked into yet. Added `dataLabel` where necessary in scan config for now, will have to update elsewhere in another PR. `dataLabel` on the expandable section seems a little weird and unnecessary but if we think it causes an accessibility issue I can add it.
![Screenshot 2024-02-13 at 1 19 22 PM](https://github.com/stackrox/stackrox/assets/61400697/7357eb56-63c8-418d-b8dc-71a64d9620fd)

Reply:
Yeah, picture above without `dataLabel` looks clearer to me than with **Profile details** here.

When there are questions in the future about a potential accessibility issues, it seems worthwhile to confirm with axe DevTools.